spring-petclinic-client/src/scripts/owner-form/owner-form.controller.js

```diff
@@ -2,9 +2,9 @@
 
 angular.module('ownerForm')
     .controller('OwnerFormController', ["$http", '$state', '$stateParams', function ($http, $state, $stateParams) {
-        var self = this;
+        const self = this;
```

Comment:
```const``` and ```let``` keywords has been introduced in ES6 (ES2015) and they are compatible with all browsers (ex: IE 10 see https://caniuse.com/#search=let)

Reply:
they are or they aren't? 
should I revert this patch?